src/components/views/elements/SyntaxHighlight.tsx

```diff
@@ -26,7 +26,7 @@ interface IProps {
 export default class SyntaxHighlight extends React.PureComponent<IProps> {
     public render(): JSX.Element {
         const { children: content, language } = this.props;
-        const highlighted = language ? hljs.highlight(content, { language }) : hljs.highlightAuto(content);
+        const highlighted = language ? hljs.highlight( { language },content) : hljs.highlightAuto(content);
```

Comment:
This doesn't look correct. Can you have another look?

Reply:
Ok I trie to make it better

Reply:
Okay I try to build this better